website/docs/cluster-management/getting-started.mdx

```diff
@@ -224,6 +224,25 @@ metadata:
     metadata.weave.works/dashboard.prometheus: https://prometheus.io/
 ```
 
+#### Specifying CAPI cluster kinds
+
+To be able to explicitly specify the type of cluster, you need to use metadata annotations using `weave.works/cluster-kind` for the annotation key as the below pattern:
+
+```
+apiVersion: gitops.weave.works/v1alpha1
+kind: GitopsCluster
+metadata:
+  annotations:
+      weave.works/cluster-kind: <CLUSTER_KIND>
+```
+where the **CLUSTER_KIND** can be one of the following supported ones:
+  - DockerCluster
+  - AWSCluster
+  - AWSManagedCluster
+  - AzureCluster
+  - AzureManagedCluster
+  - GCPCluster
```

Comment:
- LiquidMetal 
- Rancher
- Openshift
- Tanzu
- OtherOnprem

Reply:
will icons need to be added for them all?

Reply:
LiquidMetal infraProvider name is `MicrovmCluster`. Do Tanzu/Openshit/Rancher have CAPI providers?

Reply:
Tanzu is the vsphere capi provider. Openshift I don't think has one, and rancher we will likely be building a bootstrap provider. Which allows you to use the capi provider of choice but instead of bootstrapping a vanilla k8s distro, it will bootstrap the rancher rke distro.